backends/graphs/README.md

```diff
@@ -1,7 +1,9 @@
 # Graphs Backend
 
 This backend produces visual representations of a P4 program as dot files. For
-now it only supports the generation of graphs for top-level control and parser blocks.
+now it supports the generation of graphs for top-level control and parser blocks,
+generation of fullGraph, which merges graphs for top-level program blocks and
```

Comment:
what is really a "fullGraph"?

Reply:
It merges all generated boost graphs of parsers and control blocks into one graph.
This graph enables users to easily observe control flow of full program.